jwst/resample/tests/test_resample_step.py

```diff
@@ -831,9 +831,9 @@ def test_resample_undefined_variance(nircam_rate, shape):
 
 @pytest.mark.parametrize('ratio', [0.7, 1.2])
 @pytest.mark.parametrize('rotation', [0, 15, 135])
-@pytest.mark.parametrize('crpix', [(256, 488), (700, 124)])
-@pytest.mark.parametrize('crval', [(50, 77), (20, -30)])
-@pytest.mark.parametrize('shape', [(1205, 1100)])
+@pytest.mark.parametrize('crpix', [(100, 101), (101, 101)])
+@pytest.mark.parametrize('crval', [(22.01, 12), (22.15, 12.01)])
+@pytest.mark.parametrize('shape', [(10205, 10100)])
```

Comment:
This is way, way too large for a test. I think it is an indication that `crval` is wrong if you need to make output shape this large in order to get data within the output.

Reply:
I can take a look.  I have worked with these tests somewhat recently.

Reply:
I pushed up a fix that keeps the custom parameters to more reasonable values and checks that the output data is nontrivial.